frontend/BoardGUI.py

```diff
@@ -31,12 +33,22 @@ def draw_piece(self, piece, position):
         else:
             self.game.screen.blit(self.white_piece_image, position)
 
-    def getCellClicked(self):
+    def get_cell_clicked(self):
         current_cell = vec2(pg.mouse.get_pos()) // self.cell_size
         col, row = map(int, current_cell)
         left_click = pg.mouse.get_pressed()[0]
         if left_click:
-            print(self.board.board[row][col].state)
+            # print(self.backend_game.get_board(row,col), self.count)
+            self.count += 1
+
+            if not self.backend_game.placement_complete():
+                x = self.backend_game.piece_placement(row, col)
```

Comment:
Can we give this a more descriptive name?

Reply:
Sure -- is "flag" descriptive enough or do you want a more specific name? It should only be used directly afterwards.

I could also move it directly to the if-statement and not store the return value at all, if you think that'd read better.

Reply:
I'm going to change it to "flag" when I push the image/example README change, but we can always swap it later if needed.